eclair-core/src/main/scala/fr/acinq/eclair/remote/EclairInternalsSerializer.scala

```diff
@@ -128,6 +128,7 @@ object EclairInternalsSerializer {
     ("chainsHash" | bytes32) ::
       ("remoteNodeId" | publicKey) ::
       ("to" | actorRefCodec(system)) ::
+      ("replacePrevious" | bool(8)) ::
```

Comment:
Is that ok? Or do I need to do some magic for backwards-compatibility?

Reply:
We just need to make sure that we upgrade the front and back at the same time.